### ISSUE TITLE
Follow the "--title" option

### ISSUE THREAD

**Issue by s-cerevisiae** [darsain/uosc#24]
With `border=no` in mpv.conf it seems that uosc won't change it's window title according to `--title=`  config.

**Comment by darsain** [comment]
`--title` controls window title. Meaning title of the window within current window manager, as well as title of the taskbar button. Those are relevant to other apps looking at windows, and uosc doesn't touch that, they are still set to what they are supposed to be, and work as they did before.

uosc currently displays `{media-title}` in the top bar since there are 2 issues with displaying `--title` there:
1. Its pretty much just `{media-title} - mpv` and I find it redundant to display that `- mpv` here. Again, it's an info for window managers, not people using the app.
2. When I looked into potentially supporting `--title` I found out that there is no property that holds this string, or an API to get it without having to manually call an `expand-text` command on a title, which takes an unreasonable 150ms to return. That is a no-go.

So this is a wontfix, sorry if that inconveniences you in some way.
